timemachine/potentials/nonbonded.py

```diff
@@ -1,24 +1,50 @@
+import jax
+jax.config.update("jax_enable_x64", True)
```

Comment:
You probably want to remove this - let the top level code decide whether or not to enable this.

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/453/commits/0489b746e68c13000963809cdf750c936da1bbb9